src/main/java/frc/team2412/robot/sim/TalonFXSimProfile.java

```diff
@@ -0,0 +1,73 @@
+package frc.team2412.robot.sim;
+
+import static frc.team2412.robot.sim.PhysicsSim.random;
+
+import com.ctre.phoenix.motorcontrol.TalonFXSimCollection;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+
+public class TalonFXSimProfile extends SimProfile {
+    private final TalonFXSimCollection falconSimCollection;
+    private final double accelToFullTime;
+    private final double fullVel;
+    private final boolean sensorPhase;
+
+    // TODO Log pos????
+    private double pos = 0;
+    private double vel = 0;
```

Comment:
Logging here shouldn't be needed as the existing motor logging in the subsystems (if any) will already log that